voice/bxml/callbacks/conferenceMemberExit.md

```diff
@@ -1,22 +1,31 @@
 {% method %}
 ## Conference Member Exit Event
-The Conference Member Exit event is fired whenever a caller exits a [conference](../verbs/conference.md) that specified a callbackUrl.
+The Conference Member Exit event is fired whenever a caller exits a [conference](../verbs/conference.md) that specified a callbackUrl.  
+
+`<PlayAudio>` and `<SpeakSentence>` verbs can be used in BXML returned by this callback to play audio in the conference, which will be heard by all members still in the conference.
+
 ### Expected response
+
 ```http
-HTTP/1.1 204
+HTTP/1.1 200
+Content-Type: application/xml; charset=utf-8
+
+<Response>
+    <!-- <PlayAudio> or <SpeakSentence> BXML verbs to play in the conference -->
+</Response>
 ```
 
 ### Properties
 
-| Property     | Description |
-|:----------   |:------------|
-| eventType    | The event type, value is `conferenceMemberExit` |
-| conferenceId | The ID of the new conference that was created |
-| name         | The custom name used to reference this conference. This the name that you included inside the body of the [`<Conference>`](../verbs/conference.md) tag.|
-| callId       | The callId of the member that left the conference |
-| from         | The from number of the call that left the conference |
-| to           | The to number of the call that left the conference |
-| tag          | (optional) The `tag` that was set at conference creation, if any |
+| Property     | Description                                                                                                                                             |
+| :----------- | :------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| eventType    | The event type, value is `conferenceMemberExit`.                                                                                                        |
+| conferenceId | The ID of the new conference that was created.                                                                                                          |
+| name         | The custom name used to reference this conference. This the name that you included inside the body of the [`<Conference>`](../verbs/conference.md) tag. |
+| callId       | The callId of the member that left the conference.                                                                                                      |
+| from         | The from number of the call that left the conference.                                                                                                   |
+| to           | The to number of the call that left the conference.                                                                                                     |
+| tag          | (optional) The `tag` that was set at conference creation, if any.                                                                                       |
```

Comment:
I actually did this on purpose to cut down on extraneous whitespace edits in all of the PRs. We can put it back if you feel strongly but I was hoping this way would catch on lol

Reply:
Oh well... I have a VSCode plugin that does that for me, but of course I can't count on other people to use that. I can stick to less spaces in the last column to avoid the whitespace at least.